src/main/java/io/github/syst3ms/skriptparser/util/NumberUtils.java

```diff
@@ -0,0 +1,63 @@
+package io.github.syst3ms.skriptparser.util;
+
+import java.math.BigInteger;
+
+/**
+ * Utility functions for numbers that don't have to do with math.
+ */
+public class NumberUtils {
+
+	/**
+	 * Checks if a given integer is between two values, both inclusive.
+	 * @param i the value to check
+	 * @param a the lower bound
+	 * @param b the upper bound
+	 * @return whether or not the value lays in between or is equal to a and b
+	 */
+	public static boolean between(int i, int a, int b) {
```

Comment:
I don't really why this needs to be a separate function, I think just using the two comparisons is legible.